code/game/machinery/vending/marine_vending.dm

```diff
@@ -197,6 +197,7 @@
 					)
 
 	contraband = list(
+					/obj/item/ammo_magazine/smg/ppsh/extended = 20,
```

Comment:
Check how the regular SMG is balanced regarding regular and extended ammo and put in similar or inferior values. Remember it scales with population anyway.